joint_calling/vqsr.py

```diff
@@ -383,10 +248,9 @@ def main(  # pylint: disable=R0913,R0914
     else:
         hard_filtered_vcfs = gnarly_output_vcfs
     # hard_filtered_vcfs = [
```

Comment:
Are these debugging remnants (also below)?

Reply:
They are, though quite useful. I even added an option to the code to provide a Batch ID for a previous run that was run with `delete_scratch_on_exit=False`, to reuse its result; but didn't implement it for VQSR. Removed the commented code for now.